tests/sample_ipam/tests.py

```diff
@@ -14,11 +16,16 @@ class TestModel(BaseTestModel, CreateModelsMixin, TestCase):
     subnet_model = Subnet
 
 
-class TestAdmin(BaseTestAdmin, TestCase):
+class TestAdmin(BaseTestAdmin, CreateModelsMixin, TestCase):
```

Comment:
why don't you add `CreateModelsMixin` to `BaseTestAdmin` directly? It makes a lot more sense since that mixins is needed there, it can't work without it